torchrl/version.py

```diff
@@ -0,0 +1,2 @@
+__version__ = "0.0.1c0+00ddc39"
```

Comment:
is this intended?

Reply:
It's a change by microfmt. I skipped it at the beginning, but then github's CI check failed, so I added it. I added it in a separate commit to not mix it up with the main changes. I can (and I would) remove the commit if needed.

Reply:
I don't think version.py is on GitHub so let's keep it that way